scripts/createdownloadurls.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+with open("./sha256sums.txt") as fobj:
+    lines = fobj.readlines()
+
+
+def main():
+    filenames = []
+    for line in lines:
+        line = line.strip()
+        if not line:
+            continue
+        words = line.split()
+        if len(words) == 2:
+            filenames.append(words[1])
+        else:
+            print("Worng line: {0}".format(line))
+
+    filenames = list(set(filenames))
+    filenames.sort()
+    for name in filenames:
+        print("https://dev-bin.ops.securedrop.org/localwheels/{0}".format(name))
```

Comment:
As @msheiny noted, making this overridable is wise. Something like:

```
DEV_WHEELS_BUCKET_BASE_URL = "https://dev-bin.ops.securedrop.org/localwheels"
WHEELS_BUCKET_BASE_URL = os.environ.get("WHEELS_BUCKET_BASE_URL", DEV_WHEELS_BUCKET_BASE_URL)
```

at the top of the file.